fixtures/tests/access_snv/files_metadata.json

```diff
@@ -122,6 +122,7 @@
                 "labHeadName": "-",
                 "labHeadEmail": "-",
                 "piEmail": "-",
+                "sex": "M",
```

Comment:
Note that this "sex" field is required for us to be able to run the CNV pipeline, as it contains logic based on the sex of the patient.